api/analysis.go

```diff
@@ -2080,7 +2072,7 @@ func (h *AnalysisHandler) archive(ctx *gin.Context, q *gorm.DB) (err error) {
 		db = db.Where("n.IssueID = i.ID")
 		db = db.Where("i.AnalysisID", m.ID)
 		db = db.Group("i.ID")
-		summary := []ArchivedIssue{}
+		summary := []model.ArchivedIssue{}
```

Comment:
Shouldn't this be a api.ArchivedIssue?

Reply:
Seemed more appropriate to use the one from the model package because it's used directly in a query and stored on the model.

Reply:
Ah.  I misunderstood.